asserts/snap_asserts.go

```diff
@@ -111,6 +112,21 @@ func assembleSnapDeclaration(assert assertionBase) (Assertion, error) {
 	}, nil
 }
 
+// SnapFileSHA3_384 computes the SHA3-384 digest of the given snap file.
+// It also returns its size.
+func SnapFileSHA3_384(snapPath string) (digest string, size uint64, err error) {
```

Comment:
we have a place if it really useful for other files (but still assertion related) which would be asserts/digest.go here